.storybook/main.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
       },
     },
   ],
+  staticDirs: ['../src/design-tokens/tier-1-definitions'],
```

Comment:
hmmm I see something in traject that says:
```
/**
 * Although `--static-dir` is marked as deprecated. The The Chromatic CLI
 * currently pulls the staticDir from the build script, but does not support
 * the staticDirs configuration option.
 *
 * We should refrain from using the staticDirs option in this configuration until
 * https://github.com/chromaui/chromatic-cli/issues/462 is resolved.
 */
```
gonna guess this is also true here 😞